[CI] EsqlSpecForceStoredLoadingIT test {csv-spec:fuse.fuseWithRowLinearAndMultiValueGroupColumn} failing

**Build Scans:**
- [elasticsearch-intake #38490 / part3](https://gradle-enterprise.elastic.co/s/zhxsxyn33qhdu)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT" -Dtests.method="test {csv-spec:fuse.fuseWithRowLinearAndMultiValueGroupColumn}" -Dtests.seed=70E2118386F76B85 -Dtests.locale=lv-LV -Dtests.timezone=Africa/Kigali -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Afuse.fuseWithRowLinearAndMultiValueGroupColumn%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Context: {is_running=false, took=14, completion_time_in_millis=1777308090574, is_partial=false, columns=[{name=_id, type=keyword}, {name=my_score, type=double}, {name=my_fork, type=keyword}], expiration_time_in_millis=1777394490560, values=[[id_0.0, null, [foo, bar]], [id_4.0, 4.0, foo], [id_3.0, 3.0, foo], [id_2.0, 2.0, foo], [id_1.0, 1.0, foo]], start_time_in_millis=1777308090560, id=Fm9MM1hEd1FlUUw2eVpleWkzMmxLS2cdQjhSckIycWtUVTJvcmwwVERuanRWQToyNTQ2MTc=, documents_found=0, values_loaded=0}
Expected a list containing
0: expected "Line 6:3: evaluation of [FUSE LINEAR SCORE BY my_score GROUP BY my_fork] failed, treating result as null. Only first 20 failures recorded." but was <missing>
1: expected "Line 6:3: java.lang.IllegalArgumentException: group column contains multivalued entries; assigning null scores" but was <missing>
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {csv-spec:fuse.fuseWithRowLinearAndMultiValueGroupColumn}
- [main] 1 consecutive failures in step part3
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -143,9 +143,6 @@ tests:
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=search.vectors/41_knn_search_byte_quantized_bfloat16/KNN Vector similarity search only}
   issue: https://github.com/elastic/elasticsearch/issues/147251
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:fuse.fuseWithRowLinearAndMultiValueGroupColumn}
-  issue: https://github.com/elastic/elasticsearch/issues/147613
 - class: org.elasticsearch.xpack.transform.integration.TransformCrossProjectIT
   method: testUpdateTransformWithProjectRouting
   issue: https://github.com/elastic/elasticsearch/issues/147655
@@ -644,30 +641,12 @@ tests:
 - class: org.elasticsearch.xpack.stateless.StatelessMergeIT
   method: testOnlyForceMergesAreAllowedDuringRelocation
   issue: https://github.com/elastic/elasticsearch/issues/150925
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:subquery.subqueryInFromWithRenameMissingCounterFieldWithNullify}
-  issue: https://github.com/elastic/elasticsearch/issues/150943
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:subquery.subqueryInFromWithRenameChainMissingCounterFieldWithNullify}
-  issue: https://github.com/elastic/elasticsearch/issues/150963
 - class: org.elasticsearch.xpack.esql.expression.function.scalar.spatial.StBufferTests
   method: "testEvaluateInManyThreads {TestCase=geo_shape with distance[double]. #2}"
   issue: https://github.com/elastic/elasticsearch/issues/150965
 - class: org.elasticsearch.reindex.management.ReindexRelocationIT
   method: testNonSlicedRemoteReindexRelocation
   issue: https://github.com/elastic/elasticsearch/issues/150966
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:subquery.subqueryInFromWithRenameChainMissingCounterField}
-  issue: https://github.com/elastic/elasticsearch/issues/150968
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:stats.sumExpressionWithConstantThreeClausesShareOneSvPair}
-  issue: https://github.com/elastic/elasticsearch/issues/150969
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:date.dateMinusNull}
-  issue: https://github.com/elastic/elasticsearch/issues/150970
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:fuse.fuseWithLinearAndMinMax}
-  issue: https://github.com/elastic/elasticsearch/issues/150971
 - class: org.elasticsearch.xpack.esql.inference.rerank.RerankOperatorTests
   method: testInferenceFailure
   issue: https://github.com/elastic/elasticsearch/issues/150976
@@ -677,12 +656,3 @@ tests:
 - class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
   method: test {csv-spec:union_types.singleIndexIpStats}
   issue: https://github.com/elastic/elasticsearch/issues/150982
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:subquery.subqueryInFromWithRenameMissingCounterField}
-  issue: https://github.com/elastic/elasticsearch/issues/150986
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:string.in3VLWithComputedNullAsValue}
-  issue: https://github.com/elastic/elasticsearch/issues/150987
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
-  method: test {csv-spec:dense_vector-arithmetic.mulDenseVectorWithLiteral}
-  issue: https://github.com/elastic/elasticsearch/issues/150988

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/subquery.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/subquery.csv-spec
@@ -1720,6 +1720,7 @@ subqueryInFromWithRenameMissingCounterFieldWithNullify
 required_capability: subquery_in_from_command
 required_capability: subquery_in_from_command_union_types_implicit_casting_inconsistent_after_rename
 required_capability: optional_fields_nullify_tech_preview
+request_stored: skip
 
 SET unmapped_fields="nullify"\;
 FROM k8s-downsampled, (from many_numbers)

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/subquery.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/subquery.csv-spec
@@ -1703,6 +1703,7 @@ x:integer
 subqueryInFromWithRenameMissingCounterField
 required_capability: subquery_in_from_command
 required_capability: subquery_in_from_command_union_types_implicit_casting_inconsistent_after_rename
+request_stored: skip
 
 FROM k8s-downsampled, (from many_numbers)
 | KEEP network.total_bytes_in, network.eth0.tx
@@ -1738,6 +1739,7 @@ x:long | y:aggregate_metric_double
 subqueryInFromWithRenameChainMissingCounterField
 required_capability: subquery_in_from_command
 required_capability: subquery_in_from_command_union_types_implicit_casting_inconsistent_after_rename
+request_stored: skip
 
 FROM k8s, (from many_numbers)
 | KEEP network.total_bytes_in
@@ -1756,6 +1758,7 @@ subqueryInFromWithRenameChainMissingCounterFieldWithNullify
 required_capability: subquery_in_from_command
 required_capability: subquery_in_from_command_union_types_implicit_casting_inconsistent_after_rename
 required_capability: optional_fields_nullify_tech_preview
+request_stored: skip
 
 SET unmapped_fields="nullify"\;
 FROM k8s, (from many_numbers)

